src/catch2/internal/catch_test_case_info_hasher.hpp

```diff
@@ -0,0 +1,20 @@
+#ifndef CATCH_TEST_CASE_INFO_HASHER_HPP_INCLUDED
+#define CATCH_TEST_CASE_INFO_HASHER_HPP_INCLUDED
+
+#include <catch2/catch_test_case_info.hpp>
```

Comment:
Move this include down into the .cpp file and only fwd declare `TestCaseInfo` in the header.